### PR TITLE
Fix latest phpstan error on Forge

### PR DESCRIPTION
**Pull request by paulbalandan**
**Checklist:**
- [x] Securely signed commits

### CODE REVIEW

File: system/Database/Forge.php [hunk 1]
```diff
@@ -341,9 +341,10 @@ public function addKey($key, bool $primary = false, bool $unique = false)
 		else
 		{
 			$this->keys[] = $key;
-			if ($unique === true)
+
+			if ($unique)
 			{
-				$this->uniqueKeys[] = ($c = count($this->keys)) ? $c - 1 : 0;
+				$this->uniqueKeys[] = count($this->keys) - 1;
```
Comment (sfadschm) [inline system/Database/Forge.php:347]: This change seems to be right according to the phpstan interpretation.
However, looking at the old implementation, I think something like this was intended by the original contributor:
`$this->uniqueKeys[] = count($this->keys) ? (count($this->keys) - 1) : 0;`
Can't think of a shorter syntax right now.
Maybe also stricter checks should be added here.

Comment (paulbalandan) [reply system/Database/Forge.php:347]: But if you will look at the line before that, `$this->keys` is pushed with the value of `$key`, so the count will always return true and the falsy condition will never be reached, just as phpstan says.

Comment (sfadschm) [reply system/Database/Forge.php:347]: You're right, didn't think about that.

**Review by sfadschm — APPROVED** [review]

**Review by samsonasik — APPROVED** [review]

**Comment by samsonasik** [comment]
@paulbalandan thank you.

**Pull request merged**